about.html

```diff
@@ -22,24 +21,19 @@
 <h3>Applications</h3>
 
 <ul>
-<li>PcManFm-qt - File Manager</li>
-<li>Lximage-qt - Image Viewer</li>
-<li>QTerminal - Terminal Emulator</li>
-<li>Qps - Process Viewer</li>
-<li>Screengrab - Screen Capturer</li>
-<li>LXQt-archiver - Archive Manager</li>
-<li>LXQt-runner - Application launcher and calculator</li>
+<li>lxqt-runner — Launch applications and do calculations</li>
+<li>PCManFM-Qt — Browse files</li>
+<li>LXImage-Qt — View images</li>
+<li>QTerminal — Use the terminal</li>
+<li>ScreenGrab — Capture screenshots</li>
+<li>LXQt Archiver — Manage archives</li>
+<li>Qps — Manage processes</li>
 </ul>
 <br/>
 
-<p>LXQt is available for all major Linux and BSD distributions so you may just try it
-out on your system or in a VM.<br>
-More information about installing can be found in the
-<a href="https://github.com/lxqt/lxqt/wiki/">LXQt GitHub Wiki</a>.
+<p>Try LXQt in most Linux or BSD distributions.<br>
+Learn to install it in the <a href="https://github.com/lxqt/lxqt/wiki/">LXQt wiki</a>.
 </p>
```

Comment:
I like the original text more; IMHO, they have no problem. "Try" and "learn" are too imperative — "learn" might feel a little offending.

Reply:
Personally, interpreting the current text as "apologetic" sounds weird to me. I think anyone, with any native language and culture, feels the difference between "Do it!" and "This is how it can be done...".